src/components/LogPane/index.tsx

```diff
@@ -43,7 +43,11 @@ const LogPane: React.FC<LogPaneProps> = ({
         <List
           ref={listRef}
           cache={cache}
-          containerStyle={{ overflowX: "visible", overflowY: "visible" }}
+          containerStyle={{
+            overflowX: "visible",
+            overflowY: "visible",
```

Comment:
this style should be equivalent to just setting `overflow: "visible"` ([docs](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow))